subprojects/packagefiles/google-highway/meson.build

```diff
@@ -0,0 +1,155 @@
+project(
+  'highway',
+  'cpp',
+  version: '1.2.0',
+  license: 'Apache-2.0',
+)
```

Comment:
Please set a `meson_version` as well, I'd suggest `meson_version : '>= 1.3.0'`, see the following.